modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenModel.java

```diff
@@ -35,7 +36,7 @@
     public Set<String> allMandatory;
 
     public Set<String> imports = new TreeSet<String>();
-    public Boolean hasVars, emptyVars, hasMoreModels, hasEnums, isEnum, hasRequired;
+    public Boolean hasVars, emptyVars, hasMoreModels, hasEnums, isEnum, hasRequired, isArrayModel;
```

Comment:
In the next version (stable release after 2.2.0), we may want to revise the mustache tag e..g

`{{#isArrayModel}}` will become `{{#isListContainer}}{{#isModel}}`